prime-router/settings/STLTs/NM/nm-doh.yml

```diff
@@ -0,0 +1,98 @@
+- name: "nm-doh"
+  description: "New Mexico Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "NM"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "nm-doh"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/NM/NM-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "NMDOH^2.16.840.1.113883.3.5364^ISO"
```

Comment:
I noticed you're doing this for most of the STLTs. This field should only have `NMDOH` does that not work?